isis/tests/NearMsiCameraTests.cpp

```diff
@@ -0,0 +1,67 @@
+#include "MsiCamera.h"
+#include "Fixtures.h"
+#include "TestUtilities.h"
+
+#include <gtest/gtest.h>
+
+using namespace Isis;
+
+TEST_F(NearMsiCameraCube, NearMsiCameraTest) {
+  Init();
```

Comment:
See my comments on the test fixture, GTest will call this for you if you make it SetUp.